Sources/UtiliKit/Container/Transitioning/ContainerPercentDrivenInteractiveTransitioning.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  ContainerPercentDrivenInteractiveTransition2.swift
+//  UtiliKit-iOS
+//
+//  Created by Will McGinty on 1/24/19.
+//  Copyright © 2019 Bottle Rocket Studios. All rights reserved.
+//
+
+import UIKit
+
+open class ContainerPercentDrivenInteractiveTransitioner: NSObject, ContainerViewControllerInteractiveTransitioning {
+
+    // MARK: Properties
+    open var transitionAnimator: UIViewControllerAnimatedTransitioning?
+    open private(set) var transitionContext: UIViewControllerContextTransitioning?
+    private(set) var interruptibleAnimator: UIViewImplicitlyAnimating?
+    
+    // MARK: Internal State
+    open var percentComplete: CGFloat = 0.0 {
+        didSet { interruptibleAnimator?.fractionComplete = percentComplete }
+    }
+    
+    open var wantsInteractiveStart: Bool = true
+    open var completionSpeed: CGFloat = 1.0
+    open var completionCurve: UIView.AnimationCurve = .easeInOut
+    open lazy var timingCurve: UITimingCurveProvider = UICubicTimingParameters(animationCurve: completionCurve)
+    
+    // MARK: ContainerPercentDrivenInteractiveTransitioning
+    open func startInteractiveTransition(_ context: UIViewControllerContextTransitioning, using animator: UIViewControllerAnimatedTransitioning) {
+        transitionContext = context
+        transitionAnimator = animator
+        
+        guard let animator = animator.interruptibleAnimator?(using: context) else {
+            return debugPrint("In order to fully use the ContainerPercentDrivenInteractiveTransitioner, the animator must provide an object conforming to UIViewImplicitlAnimating.")
```

Comment:
Minor typo for "UIViewImplicitlAnimating" at the end. We also might want to use the classes themselves to support any potential renaming from my comment above.
```swift
return debugPrint("In order to fully use the \(ContainerPercentDrivenInteractiveTransitioner.self), the animator must provide an object conforming to \(UIViewImplicitlyAnimating.self).")
```

Reply:
Done.